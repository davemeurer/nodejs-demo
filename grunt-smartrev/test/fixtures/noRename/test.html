<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
    <link rel="stylesheet" href="test.css">
</head>
<body>
    <a href="logo.png"><img src="logo.png"></a>
    <script src="test.js"></script>
</body>
</html>