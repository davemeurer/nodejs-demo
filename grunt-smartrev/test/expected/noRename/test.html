<!DOCTYPE html>
<html>
<head>
    <title>Test</title>
    <link rel="stylesheet" href="test.css" />
</head>
<body>
    <a href="logo.0f278ffd46a4687731ccad34403db8f9.png"><img src="logo.0f278ffd46a4687731ccad34403db8f9.png" /></a>
    <script src="test.f7ea418b32c0cc27e550d3373796660d.js"></script>
</body>
</html>